docs/contributing.md

```diff
@@ -185,88 +222,102 @@ If you are not currently in the `main` branch, run the following command to retu
+
 Create a new branch where you will work on the issue. The branch name should include the issue number. For example, to create a new branch for issue 15 and change into it:
 
 ```bash
 git checkout -b <new-branch-name>-15
 ```
 
-#### **2.2.b Working on an issue(2): Prepare your changes to push to your repository**
+## 3.3 Make Changes
+
+Make changes to fix the issue.
+
+## 3.4 Pull to Get Most Recent Code
 
-Once you are done with the work on your issue you will push it to your repository.  Before you can push your work to your repository, you will stage and commit your changes.  These two commands are similar to the save command used in other programs.
+You can probably skip this if you fix the issue on the same day that you pulled the code.
 
-**Note:** If you are using Visual studios code you can use the Git graphical user interface to stage your changes. For instructions check out the [Git Gui Wiki](https://github.com/hackforla/website/wiki/Using-Git-GUI-(Graphical-user-Interface)-in-Visual-Studios-Code).
+```bash
+git pull
+```
 
-**Note:** Remember to run the `precommit-checks.sh` script before each commit until it can be integrated.
+**Note:** If you are using Visual studios code you can use the Git graphical user interface to stage your changes. For instructions check out the [Git Gui Wiki](<https://github.com/hackforla/website/wiki/Using-Git-GUI-(Graphical-user-Interface)-in-Visual-Studios-Code>).
 
-##### **i. Prepare repo changes (1): Use the `git add` command to stage your changes.**
+## 3.5 Add Changed Files to Staging
 
 **Make sure you are on your issue branch (instead of `main`)**
 
-This command prepares your changes before you commit them. You can stage files one at a time using the filename.
+```bash
+git branch
+```
+
+You must add your files to the staging area before you can commit (save them to git).
 
 Run this command if you want to **add changes from a specific file to your commit record**:
+
 ```bash
 git add “filename.ext”
 ```
 
 Run this command if you want to **add all changes to all file(s) to your commit record**:
+
 ```bash
 git add .
 ```
 
-##### **ii. Prepare repo changes (2): Use the `git status` command to see what files are staged.**
+## 3.6 Check git status
 
 This command will list the files that have been staged with green text. These are the files that will be committed (saved) when you run the next command, `git commit`. Please be sure all your staged changes are relevant to the issue you are working on. If you accidentally included unrelated changes, please unstage them before making this commit, and then make a new commit for the unrelated changes. (The commands for unstaging commits are provided in the output of your `git status` command.)
 
 ```bash
 git status
 ```
 
-##### **iii. Prepare repo changes (3): Use the `git reset HEAD <filename>` command to remove a staged file.**
+## 3.7 Remove Files that you don't want Staged
```

Comment:
`## 3.7 Remove Files That You Don't Want Staged`

Reply:
Done